addon/mixins/run.js

```diff
@@ -41,14 +41,6 @@ export function pollTaskFor(label) {
  @public
  */
 export default Mixin.create({
-  init() {
-    this._super(...arguments);
-
-    this._pendingTimers = [];
```

Comment:
Same as above, we need to shape the object by setting these properties to undefined.